### ISSUE TITLE
Javaagent doesn't work on linux

### ISSUE THREAD

**Issue by jgogstad** [Microsoft/ApplicationInsights-Java#291]
The javaagent doesn't run on linux. Applications doesn't start and quits immediately.

Testcase available here: [foo.zip](https://github.com/Microsoft/ApplicationInsights-Java/files/383393/foo.zip)

The attached zip contains the output from `mvn archetype:generate` with defaults everything. I configured maven-jar-plugin to use the proper main class. Run with `java -jar agentbug/target/agentbug-1.0-SNAPSHOT.jar`

Tests (OpenJDK 8 on Alpine):

    docker@docker1:~$ docker run --rm -v /c/Users/gmjgo/Downloads/foo/:/foo java:8-jre-alpine java -javaagent:foo/applicationinsights-agent-1.0.3.jar -jar foo/agentbug/target/agentbug-1.0-SNAPSHOT.jar
    docker@docker1:~$ docker run --rm -v /c/Users/gmjgo/Downloads/foo/:/foo java:8-jre-alpine java -jar foo/agentbug/target/agentbug-1.0-SNAPSHOT.jar
    Hello World!
    docker@docker1:~$

Doesn't work with Oracle either:

    docker@docker1:~$ docker run --rm -v /c/Users/gmjgo/Downloads/foo/:/foo lwieske/java-8 java -javaagent:foo/applicationinsights-agent-1.0.3.jar -jar foo/agentbug/target/agentbug-1.0-SNAPSHOT.jar
    docker@docker1:~$

**Comment by OriYosefiMSFT** [comment]
Thanks for reporting @jgogstad. I will look into it.

Ori

**Comment by OriYosefiMSFT** [comment]
I believe this is fixed by [#287](https://github.com/Microsoft/ApplicationInsights-Java/pull/287). Will do some more validation before closing the issue

**Comment by jgogstad** [comment]
Testet OK on current master branch

```
docker@docker1:~$ docker run --rm -v /c/Users/gmjgo/foo/bar:/foo java:8-jre-alpine java -javaagent:foo/applicationinsights-agent-1.0.5-SNAPSHOT.jar -jar foo/agentbug/target/agentbug-1.0-SNAPSHOT.jar
AI-Agent: INFO 02-08-2016 09:11, 1: Agent is up
Hello World!
docker@docker1:~$
```

When do you expect the next official release?

**Comment by gupele** [comment]
We are aiming to early next week

**Comment by OriYosefiMSFT** [comment]
Many thanks for verifying this (and also for your other contributions). It is much appreciated.
As @gupele mentioned, we are hoping to release an official build early next week.

**Comment by OriYosefiMSFT** [comment]
Fixed by [287](https://github.com/Microsoft/ApplicationInsights-Java/pull/287) will be included in 1.0.5
